tfjs-backend-webgpu/src/kernels/Step.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import {KernelConfig, KernelFunc, Step, StepAttrs, TensorInfo, UnaryInputs} from '@tensorflow/tfjs-core';
+
+import {WebGPUBackend} from '../backend_webgpu';
+import {UnaryOpType} from '../unary_op_util';
+import {UnaryOpProgram} from '../unary_op_webgpu';
+
+export function step(
+    {inputs, attrs, backend}:
+        {inputs: UnaryInputs, attrs: StepAttrs, backend: WebGPUBackend}):
+    TensorInfo {
+  const {x} = inputs;
+  const program =
+      new UnaryOpProgram(x.shape, UnaryOpType.STEP, 'stepAlpha : f32,');
```

Comment:
Currently, for additional uniforms in unary ops, one example is [LeakyRelu](https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-webgpu/src/kernels/LeakyRelu.ts#L34). Should we keep them consistent in usage?

Reply:
I think we can unify the LeakyRelu to this solution.